cmd/spiffe-aws-assume-role/cli/cli.go

```diff
@@ -40,24 +52,28 @@ func (c *CredentialsCmd) Run() error {
 	return err
 }
 
-var CLI struct {
+type CLI struct {
 	Credentials CredentialsCmd `kong:"cmd,help:'print credentials in a format usable as an AWS credentials_process'"`
 }
 
-func Run(args []string) error {
-	p, err := kong.New(&CLI)
+func Run(args []string, context *CliContext) error {
```

Comment:
context type arguments should be the first argument to a function in golang

Reply:
For my own edification, this is mentioned here:

https://golang.org/pkg/context/